core-java-modules/core-java-date-operations-3/src/main/java/com/baeldung/utiltosqldate/UnitToSqlDateUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.utiltosqldate;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.TimeZone;
+
+public class UnitToSqlDateUtils {
```

Comment:
I think you meant to name this class `UtilToSqlDateUtils`?

```suggestion
public class UtilToSqlDateUtils {
```